schedule.html

```diff
@@ -0,0 +1,322 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <title>
+      W3C Workshop on Web &amp; Virtual Reality
+    </title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+    <p>
+      <a href="https://www.w3.org/"><img alt="W3C" src=
+      "https://www.w3.org/Icons/w3c_home" height="48" width="72"></a>
+    </p>
+    <h1 id="top">
+      W3C Workshop on Web &amp; Virtual Reality
+    </h1>
+    <p>
+      October 19-20, 2016; San Jose, CA, USA
+    </p>
+    <ul class="menu">
```

Comment:
Sounds like a great enhancement. I'd say this is not on the critical path, so don't spend too much time on this unless you have plenty :-)

(That said, Dom would be probably happy to steam your pimped up W3C workshop template for later use ;-))